archey/test/test_archey_gpu.py

```diff
@@ -3,12 +3,19 @@
 import unittest
 from unittest.mock import patch
 
+from subprocess import CalledProcessError
+
 from archey.entries.gpu import GPU
 
 
 class TestGPUEntry(unittest.TestCase):
     """Here, we mock the `check_output` call to `lspci` to test the logic"""
-
+    @patch(
+        'archey.configuration.Configuration.get',
+        side_effect=[
+            {'max_count': None}
+        ]
+    )
```

Comment:
Unfortunately the configuration options I added mean we now need to patch all of our tests' `Configuration` singletons...